pkgs/desktops/deepin/dde-polkit-agent/dde-polkit-agent.plugins-dir.patch

```diff
@@ -0,0 +1,27 @@
+From 69d0fb1841d2069af544d2dda0d6ba980dc0106c Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jos=C3=A9=20Romildo=20Malaquias?= <malaquias@gmail.com>
+Date: Fri, 19 Apr 2019 22:01:16 -0300
+Subject: [PATCH] deepin.dde-polkit-agent: use an environment variable to find
+ plugins
+
+---
+ pluginmanager.cpp | 3 ++-
+ 1 file changed, 2 insertions(+), 1 deletion(-)
+
+diff --git a/pluginmanager.cpp b/pluginmanager.cpp
+index 0c03237..9c46eb9 100644
+--- a/pluginmanager.cpp
++++ b/pluginmanager.cpp
+@@ -34,7 +34,8 @@ QList<QButtonGroup*> PluginManager::reduceGetOptions(const QString &actionID)
+ void PluginManager::load()
+ {
+ 
+-    QDir dir("/usr/lib/polkit-1-dde/plugins/");
++    QDir dir(QProcessEnvironment::systemEnvironment().value("DDE_POLKIT_PLUGINS_DIR", "/usr/lib/polkit-1-dde/plugins/"));
```

Comment:
Can you give me an example or a source where DDE ships a plugin like this for polkit?

Reply:
> Can you give me an example or a source where DDE ships a plugin like this for polkit?

`lib/polkit-1-dde/plugins/libdpa-ext-gnomekeyring.so` from [`dpa-ext-gnomekeyring`](https://github.com/linuxdeepin/dpa-ext-gnomekeyring)